pkg/infra/main_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"flag"
+	"infra/logging"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
This is unused as of now, main testing requires more tests to be provided in separate functions that will use main test body as closures.